packages/elastic-apm-synthtrace/src/cli.ts

```diff
@@ -6,4 +6,4 @@
  * Side Public License, v 1.
  */
 
-export { runSynthtrace } from './scripts/run_synthtrace';
+export { runSynthtrace } from './cli/run_synthtrace';
```

Comment:
"scripts" directories are used to store tooling files, which are only necessary for local development and not a part of package source code. I hope this rename is acceptable